server/views/components/instructions/template.njk

```diff
@@ -23,4 +23,9 @@
 
 {% block javascripts %}
     <script src="/assets/instructions.js"></script>
+    {% if params.inGridRow %}
```

Comment:
Maybe I should add a new parameter for this?

Reply:
yeah, that would be nice, not sure what inGridRow is?